extensions/ui/org.eclipse.smarthome.ui.paper/bower.json

```diff
@@ -26,7 +26,10 @@
     "bootstrap": "~3.3.2",
     "tinycolor": "~1.1.2",
     "masonry": "~3.3.1",
-    "roboto-fontface": "~0.4.3"
+    "roboto-fontface": "~0.4.3",
+    "jquery-ui": "~1.11.0",
```

Comment:
I have created CQ 10562 for JQuery version 1.11.2. Could you please change the PR to this version?